blender/arm/material/make_voxel.py

```diff
@@ -1,29 +1,253 @@
 import bpy
-
 import arm.utils
 import arm.assets as assets
+import arm.material.cycles as cycles
 import arm.material.mat_state as mat_state
-
-if arm.is_reload(__name__):
-    arm.utils = arm.reload_module(arm.utils)
-    assets = arm.reload_module(assets)
-    mat_state = arm.reload_module(mat_state)
-else:
-    arm.enable_reload(__name__)
-
+import arm.material.mat_utils as mat_utils
+import arm.material.make_particle as make_particle
+import arm.make_state as state

```

Comment:
Looks like the module reloading got lost here

Reply:
Ok